app/javascript/selectors/detailSelector.test.js

```diff
@@ -263,4 +277,82 @@ describe('selectors', () => {
       expect(checkEditDisable(state)).toEqual(true);
     });
   });
+
+  describe('#disableRolesDropDown', () => {
+    it('return true when editable is false', () => {
+      const state = {
```

Comment:
To minimize lines added to this large file, let's pull this const declaration up into a function in the describe block, which takes as one parameter the value of 'editable'.  I'm doing that in may PR, and it saved dozens of lines and looks cleaner as well, I think.
```suggestion
      const getFetchDetailEditableState = editable => {
      return {
        fetchDetails: {
          details: {
            XHRStatus: 'ready',
            records: {
              editable: editable,
              user: {},
            },
          },
        },
      };
    };
    ...
    const state = getFetchDetailEditableState(false);  // or null or true or undefined
```